modules/core/src/main/scala/org/scalasteward/core/sbt/SbtAlg.scala

```diff
@@ -91,17 +92,22 @@ object SbtAlg {
 
       override def getDependencies(repo: Repo): F[List[Dependency]] =
         for {
-          repoDir <- workspaceAlg.repoDir(repo)
-          cmd = sbtCmd(List(libraryDependenciesAsJson, reloadPlugins, libraryDependenciesAsJson))
-          lines <- exec(cmd, repoDir)
+          originalDependencies <- getOriginalDependencies(repo)
           maybeSbtVersion <- getSbtVersion(repo)
           maybeSbtDependency = maybeSbtVersion.flatMap(sbtDependency)
           maybeScalafmtVersion <- scalafmtAlg.getScalafmtVersion(repo)
           maybeScalafmtDependency = maybeScalafmtVersion.map(
             scalafmtDependency(defaultScalaBinaryVersion)
           )
         } yield (maybeSbtDependency.toList ++ maybeScalafmtDependency.toList ++
-          parser.parseDependencies(lines)).distinct
+          originalDependencies).distinct
+
+      def getOriginalDependencies(repo: Repo): F[List[Dependency]] =
+        for {
+          repoDir <- workspaceAlg.repoDir(repo)
+          cmd = sbtCmd(List(libraryDependenciesAsJson, reloadPlugins, libraryDependenciesAsJson))
```

Comment:
Need investigation) It seems that `libraryDependenciesAsJson` command does not include compiler plugin.
Therefore `Update` for `kind-projector` is not found, since it is supposed to be used as `addCompilerPlugin("org.spire-math" %% "kind-projector" % "0.9.10")`.

Reply:
`show libraryDependencies` includes compiler plugins,
```
sbt> show libraryDependencies
[info] project1 / libraryDependencies
[info]  List(org.scala-lang:scala-library:2.12.7, org.spire-math:kind-projector:0.9.10:plugin->default(compile))
```

but `show libraryDependenciesAsJson` do not.

```
sbt> show libraryDependenciesAsJson
[info] project1 / libraryDependenciesAsJson
[info]  [ { "groupId": "org.scala-lang", "artifactId": "scala-library", "artifactIdCross": "scala-library", "version": "2.12.7" } ]
```

Reply:
`libraryDependenciesAsJson` is `StewardPlugin`'s feature.

This issue is fixed in https://github.com/fthomas/scala-steward/pull/676/commits/a02f3fffa638fe122e84e87b74dbfc5d663d6e02 and now compiler plugins get updated.
Example https://github.com/exoego/scala-steward-test-project/pull/91